lighthouse-extension/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "lighthouse-extension",
   "private": true,
   "engines": {
-    "node": ">=0.8.0"
+    "node": ">=6"
```

Comment:
yeah, I don't know how much we care about the precision on this. We're going to only be testing 6.9.1, so it makes sense that we shouldn't say we work in case important bugs were fixed before that...but on the other hand we're not testing with the latest v6, so it's also possible that some bug was fixed *after* 6.9.1 that materially affects execution...